ocs_samples/basic_samples/SDS/DotNet/SdsClientLibraries/SdsClientLibraries/Program.cs

```diff
@@ -68,7 +68,7 @@ public static async Task<bool> MainAsync(bool test = false)
             Console.WriteLine(@"        \/      \/    \/  \/         \/        \/            ");
```

Comment:
For some reason it's not letting me comment on the line I want to comment on so I'm gonna put this as close as possible... Remove lines 22 and 23, they are already on lines 15 and 16. I know this is not your doing but let's make it perfect... Thank you :)

Reply:
60 and 61 were the top of the "SDS.net" console output. but above them is two lines of blank space. So one of those is removed.

Reply:
Well I think they moved up once you removed the ones above so good... thx